Resources/public/js/views/services/ez-roleserversideviewservice.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-roleserversideviewservice', function (Y) {
+    "use strict";
+    /**
+     * Provides the role server side view service class
+     *
+     * @method ez-roleserversideviewservice
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The Role Server Side View Service class.
+     *
+     * @namespace eZ
+     * @class RoleServerSideViewService
+     * @constructor
+     * @extends eZ.ServerSideViewService
+     */
+    Y.eZ.RoleServerSideViewService = Y.Base.create('roleServerSideViewService', Y.eZ.ServerSideViewService, [], {
+        initializer: function () {
+            this.on('*:contentDiscover', function (e) {
```

Comment:
actually I guess @glye followed the section view service where it's done like that and that's OK I think.